### DISCUSSION TITLE
Tailwind v4: Border double

### DISCUSSION THREAD

**Discussion by itsezc** [tailwindlabs/tailwindcss#13212] [Ideas]
The class `border-double` does not work as intended

<img width="398" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/33750251/44177f9b-82b5-425a-ada3-97ffa971b69f">

The class is applied in the browser

<img width="362" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/33750251/2fcf77e0-4018-4188-91a0-6b36917a2d2d">

Using Tailwind v4 alpha 8, Safari 17.4

**Comment by alex-krasikau** [comment]
Hey @itsezc!

The `double` style shows two straight lines totaling the pixel amount set as `border-width`. To display the double border, use a `border-width` of at least 2px. Check out this [example](https://play.tailwindcss.com/C5Fvk5u24M) with various options.
